internal/sink/sink.go

```diff
@@ -77,13 +74,19 @@ type Kafka struct {
 	CommonWebhook
 }
 
-func NewSink(c Config, logger *zap.Logger, listener ancla.Register) Sink {
+func NewSink(c Config, logger *zap.Logger, listener ancla.Register) (Sink, error) {
 	switch l := listener.(type) {
 	case *ancla.RegistryV1:
 		v1 := &WebhookV1{}
 		v1.Update(c, logger, l.Registration.Config.AlternativeURLs, l.GetId(), l.Registration.FailureURL, l.Registration.Config.ReceiverURL)
-		return v1
+		return v1, nil
 	case *ancla.RegistryV2:
+		if len(l.Registration.Kafkas) == 0 && len(l.Registration.Webhooks) == 0 {
```

Comment:
would it make sense to add this to the validation portion of web hook-schema? just thinking about ways to try and clean up the this switch statement